docs/tutorial/work-pools.md

```diff
@@ -0,0 +1,306 @@
+│                                                                                                          │
+│     Updates in Prefect workspace                                                                         │
+│                                                                                                          │
+│         - Create GCP credentials block my--pool-push-pool-credentials to store the service account key   │
+│                                                                                                          │
+╰──────────────────────────────────────────────────────────────────────────────────────────────────────────╯
+Proceed with infrastructure provisioning? [y/n]: y
+Activating Cloud Run API
+Activating Artifact Registry API
+Creating Artifact Registry repository
+Configuring authentication to Artifact Registry
+Setting default Docker build namespace
+Creating service account
+Assigning roles to service account
+Creating service account key
+Creating GCP credentials block
+Provisioning Infrastructure ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 100% 0:00:00
+Infrastructure successfully provisioned!
+Created work pool 'my-cloud-run-pool'!
+```
+
+</div>
+
+After infrastructure provisioning completes, you will be logged into your new Artifact Registry repository and the default Docker build namespace will be set to the URL of the repository.
+
+While the default namespace is set, any images you build without specifying a registry or username/organization will be pushed to the repository.
+
+To take advantage of this functionality, you can write your deploy scripts like this:
+
+```python hl_lines="14" title="example_deploy_script.py"
+from prefect import flow                                                       
+from prefect.deployments import DeploymentImage                                
+
+
+@flow(log_prints=True)
+def my_flow(name: str = "world"):
+    print(f"Hello {name}! I'm a flow running on Cloud Run!")
+
+
+if __name__ == "__main__":                                                     
+    my_flow.deploy(                                                            
+        name="my-deployment",
+        work_pool_name="above-ground",
+        image=DeploymentImage(
+            name="my-image:latest",
+            platform="linux/amd64",
+        )
+    )
+```
+
+This will build an image with the tag `<region>-docker.pkg.dev/<project>/<repository-name>/my-image:latest` and push it to the repository.
+
+See the [Push Work Pool guide](/guides/push-work-pools/) for more details and example commands for each cloud providers.
+
+## Next step
+
+Congratulations! You've learned how to use work pools to deploy flows to remote infrastructure.
+
+Learn how to use work pools that rely on a worker and build Docker images so that your flows can run in other container-based infrastructure in the [next section of the tutorial](/tutorial/workers/).
```

Comment:
It's also possible the reader could stop here if this setup is what they need. We could include some reasons why they would want to use a worker setup. I think the primary reason will be because they run their own Prefect API server or because they want to execute flows in a Kubernetes cluster.